backend/src/apiserver/server/list_request_util.go

```diff
@@ -206,3 +214,16 @@ func validatedListOptions(listable list.Listable, pageToken string, pageSize int
 
 	return opts, nil
 }
+
+func transformJSONForBackwardCompatibility(jsonStr string) (string, error) {
```

Comment:
Could you add a docstring here? What backwards compatibility is this referring to?